R/addin.R

```diff
@@ -0,0 +1,32 @@
+#' Add minimal RStudio Addin binding
+#'
+#' This function helps you add a minimal
+#' [RStudio Addin](https://rstudio.github.io/rstudioaddins/) binding to
+#' `inst/rstudio/addins.dcf`.
+#'
+#' @param addin Name of the addin function, which should be defined in the
+#' `R` folder.
+#' @inheritParams use_template
+#'
+#' @export
+use_addin <- function(addin = "new_addin", open = interactive()) {
+  addin_dcf_path <- "inst/rstudio/addins.dcf"
+
+  if (!file_exists(addin_dcf_path)) {
+    create_directory("inst/rstudio")
+    file_create(addin_dcf_path)
+    ui_done("Creating {ui_path(addin_dcf_path)}")
+  }
```

Comment:
Shouldn't all of the above be happening with the current project? I don't understand why these paths aren't using `proj_path()` but the `edit_file()` call below does use it.

Reply:
I think they should! Sorry about that. I will get it fixed shortly.